samples/openthread/ncp/README.rst

```diff
@@ -67,7 +67,7 @@ User interface
 
 All the interactions with the application are handled using serial communication.
 
-For the interaction with the application, this sample uses `wpantund`_ process with wpanctl commands.
+For the interaction with the application, this sample uses :ref:`ug_thread_tools_wpantund` process with ``wpanctl`` commands.
```

Comment:
Should we normally use monospaced ``wpanctl`` and ``wpantund`` along the docs? Why just here?

Reply:
I thought about this, but these are names of the tools, aren't they? The monospaced formatting is to be used only for code within text.

Reply:
Actually they are both names and commands :)

Reply:
But is ``wpanctl`` alone a command? For me, in the context of this sentence, it is a name.

Reply:
Yeah, in this sentence it's a name.